WcaOnRails/app/models/regulation.rb

```diff
@@ -0,0 +1,16 @@
+class Regulation < SimpleDelegator
+  begin
+    REGULATIONS = JSON.parse(File.read(Rails.root.to_s + "/app/views/search_results/wca-regulations.json")).freeze
```

Comment:
So this should be `"#{Rails.root}/app/views/regulations/wca-regulations.json` after #747, right?

The exception catching shouldn't be necessary or is there any risk?